src/services/GraphDataSource.ts

```diff
@@ -216,32 +216,46 @@ export default class GraphDataSource {
 
   // Some helpers
 
-  public fetchForApp = (duration: DurationInSeconds, namespace: string, app: string) => {
+  public fetchForApp = (duration: DurationInSeconds, namespace: string, app: string, injectServiceNodes?: boolean) => {
```

Comment:
In general, I really dislike method flags, as they hide two methods/two different implementations. Not sure how to make this better, though, so just noting.

Reply:
life is full of compromise.